.travis.yml

```diff
@@ -7,21 +7,29 @@ git:
 sudo: required
 dist: trusty
 
+nodejs:
+    - "4"
+
 notifications:
   email: false
 
 branches:
   only:
     - master
+    # enable building tags
+    - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 
 cache:
+  yarn: true
```

Comment:
Ah I guess this is https://blog.travis-ci.com/2016-11-21-travis-ci-now-supports-yarn - is yarn worth it locally?